packages/specs/schema-async-api/readme.md

```
@@ -0,0 +1,78 @@
+
+## Installation
+
+You can get the latest release and the type definitions using npm:
+
+```bash
+npm install --save @tsed/schema-async-api
+```
+
+## Contributors
+
+<a href="https://github.com/tsedio/tsed/graphs/contributors"><img src="https://opencollective.com/tsed/contributors.svg?width=890" /></a>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: a

(MD033, no-inline-html)